bin/check-licenses.js

```diff
@@ -32,11 +33,16 @@ async function main() {
     )
     throw new Error('dependencies mismatch')
   }
+
   console.log(`\n✅ All dependencies listed in ${LICENSE_FILE}`)
+
+  if (hasColumnCountMismatch) {
+    throw Error('The CSV format is incorrect. You may have an extra comma in a copyright.')
+  }
 }
 
 async function findPackageJsonPaths() {
-  const {stdout} = await exec('find . -path "*/node_modules/*" -prune -o -name "package.json" -print')
+  const { stdout } = await exec('find . -path "*/node_modules/*" -prune -o -name "package.json" -print')
```

Comment:
Why this change?
Lint config we are using usually does not have spaces (it would be rule `object-curly-spacing` to `never` with eslint) neither does it seem set in the ongoing eslint migration.
```suggestion
  const {stdout} = await exec('find . -path "*/node_modules/*" -prune -o -name "package.json" -print')
```

Reply:
Oops, I wasn't using Prettier for JS files.